### ISSUE TITLE
Maintainers

### ISSUE THREAD

**Issue by will-ockmore-axomic** [syrusakbary/snapshottest#118]
This library appears to be stale at the minute - would @syrusakbary be interested in adding further maintainers, to work on the open PRs and issues?

**Comment by paulmelnikow** [comment]
I make use of this library and I'd be interested in helping!

**Comment by versae** [comment]
I'd be great to see some of those PRs being merged at some point.

**Comment by gustavopp93** [comment]
i'd like to help too :)

**Comment by chenliu0831** [comment]
We are using this as well. Happy to help. 

> I make use of this library and I'd be interested in helping!

Hi Paul!

**Comment by paulmelnikow** [comment]
> Hi Paul!

😁 Hi Peter! 👋

**Comment by paulmelnikow** [comment]
I wrote Syrus on May 4. He wrote back a few hours later and I responded, but I haven't heard anything since then. I'll follow up and see what he says!

**Comment by syrusakbary** [comment]
Happy to add maintainers!

Thanks @paulmelnikow for reaching me personally. I just added you to the repo :)

**Comment by paulmelnikow** [comment]
I noticed there are some alternate projects mentioned in #115. I hadn't noticed those before and at some point may investigate to see if one of those is farther along.

Meanwhile, I'm happy to try to review and merge PRs. I merged the most popular unanswered PR #66 and left comments on a few more. I suggest we get a beta release out within a few days with the changes from #66, to get our release pipeline flowing. In short order we should also merge #112 or something similar.

We should get the version number out of 0.x territory. Maybe we publish the next release as 1.0.0b0, and once we've merged #112, are confident in the beta, and have made all the breaking changes we want to make, in the short term, release it as 1.0.0?
